packages/babel-plugin-jest-hoist/src/index.js

```diff
@@ -110,7 +110,7 @@ FUNCTIONS.mock = args => {
       if (!found) {
         invariant(
           (scope.hasGlobal(name) && WHITELISTED_IDENTIFIERS[name]) ||
-            /^mock/.test(name) ||
+            /^[Mm]ock/.test(name) ||
```

Comment:
`/^mock/i`? `MOCK_THING` should be fine as well, IMO.

Reply:
Should also update the message on line 126

Reply:
Sure.